cli/flags.rs

```diff
@@ -172,8 +174,18 @@ To get help on the another subcommands (run in this case):
     ).arg(
       Arg::with_name("reload")
         .short("r")
+        .min_values(0)
+        .multiple(true)
+        .takes_value(true)
+        .use_delimiter(true)
+        .require_equals(true)
```

Comment:
Are those required? Would be nice to do `--reload https://deno.land/std/fs/utils.ts https://deno.land/std/fmt/colors.ts`